processor/metricstransformprocessor/README.md

```diff
@@ -17,12 +17,15 @@ The metrics transform processor can be used to rename metrics, labels, or label
   - Aggregation_type: sum, mean, max
 - Add label to an existing metric
 - When adding or updating a label value, specify `{{version}}` to include the application version number
+- Convert resource attributes to metric labels
 
 ## Configuration
 ```yaml
 # transforms is a list of transformations with each element transforming a metric selected by metric name
 transforms:
   # name is used to match with the metric to operate on. This implementation doesn’t utilize the filtermetric’s MatchProperties struct because it doesn’t match well with what I need at this phase. All is needed for this processor at this stage is a single name string that can be used to match with selected metrics. The list of metric names and the match type in the filtermetric’s MatchProperties struct are unnecessary. Also, based on the issue about improving filtering configuration, it seems like this struct is subject to be slightly modified.
+
+  # new addition: now it accepts `all_metrics` as the value of 'metric_name' field. This batch operation applies changes to all the metrics. However, currently, it only works and is tested for the 'convert_resource_attributes_to_labels' operation. For other operations (e.g. update_label, add_label), it may not work properly. But, it shouldn't break existing user experience.
```

Comment:
Instead of adding this in the README, capture it in the PR description and file an issue to fix for other operations in the future.

Reply:
Created an issue: #1342